strawberryfields/backends/gaussianbackend/backend.py

```diff
@@ -102,24 +117,35 @@ def measure_homodyne(self, phi, mode, select=None, **kwargs):
         Returns:
             float: measured value
         """
-        eps = kwargs.get('eps', 0.0002)
```

Comment:
@nquesada since the same tolerance was used internally in gaussian backend, I've removed this explicit usage here